_faqs/M1Workaround.md

```diff
@@ -0,0 +1,35 @@
+---
+title: How to Install BciPy on Mac M1 Chip 
+---
+
+If you are using a computer with the Mac M1 chip, you will run into issues with installing some packages of BciPy including numpy and psychopy. 
+
+The best workaround for this is to use the Rosetta Terminal. 
+
+## To create the Rosetta Terminal: 
+1. Duplicate the Apple terminal and rename one of the terminals "Rosetta Terminal" 
+2. Right click on the terminal you have renamed and click "Get info"
+From the "Get info" menu, check the box that says "Open using Rosetta" 
+
+Launch the Rosetta Terminal and use only the Rosetta Terminal for all the following steps/any usage of BciPy. 
+
+
+## To Install BciPy (within the Rosetta Terminal)
+1. Git clone https://github.com/BciPy/BciPy.git
+2. Run the following command in your terminal:
+    > `arch -arm64 brew install python@3.9`
+3. Create a virtual environment using python 3.9:
+    > `python3.9 -m venv venv`
```

Comment:
Technically, we don't support 3.9. I wonder if that caused some issues with install at all. https://github.com/CAMBI-tech/BciPy#dependencies